### PR TITLE
auquantoolbox: init at 2.0.8

### PR DESCRIPTION
**Pull request by pallavagarwal07**
###### Motivation for this change
I need this package for development.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/pandas-datareader/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    extension = "tar.gz";
+    sha256 = "1j8qzzpnk8h8annyz5a3iwx2yzw007i75fk9b0rz93151i4cl8vm";
+  };
+
+  disabled = isPyPy;
+  propagatedBuildInputs = [
+    pandas requests requests-ftp requests-file python-dateutil
+    pytz numpy urllib3 idna chardet certifi six
+  ];
+  doCheck = false;
```
Comment (FRidh) [inline pkgs/development/python-modules/pandas-datareader/default.nix]: when disabling tests or builds a comment with a reason should be left

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -11702,6 +11702,14 @@ in {
     blas = pkgs.openblasCompat;
   };
 
+  auquantoolbox = callPackage ../development/python-modules/auquantoolbox { };
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: try not to make the ordering even worse

**Review by FRidh — CHANGES_REQUESTED** [review]
- `name` should be removed
- `meta.maintainers` is missing
- `meta.license` is missing
- `requests` is most likely a `propagatedBuildInput`

**Comment by pallavagarwal07** [comment]
@FRidh  Fixed most of the things. I don't know who to put as maintainer.

**Comment by prusnak** [comment]
@pallavagarwal07 Can you please rebase your PR?

**Pull request closed without merging**